robot/Cumulus/resources/locators_48.py

```diff
@@ -232,6 +233,9 @@
     'object_manager':{
         'button':'//input[@title="{}"]',
         },
+    'schedule_payment':{
+        'field_input':'//input[@value= "{}"]',
```

Comment:
line 168 and line 237 are same locators, also we already have a //input[contains(@value,"{}")] locator in line 58, does that not work here?

Reply:
@rjanjanam this one?

Reply:
I wanted to have this field to uniquely identify the input_field for the page Schedule Payment and was following the page locators file standards existing.

Reply:
we dont hardcode values in locators if the locator can be reused and name it something common so that it can be used across all pages. We should avoid having duplicate locators when there is no need

Reply:
Updated to use existing locator